api.yaml

```diff
@@ -362,11 +362,70 @@ paths:
           - "responsevalidation"
           - "lambda"
         lambda:
-          arn: "fetchByArnID"
+          arn: "fetchByResourceID"
           async: false
           request: >
             {
-              "arnid": "#!.Request.URL.arnid!#",
+              "resourceid": "#!.Request.URL.resourceid!#",
+              "time": "#!index .Request.Query.time 0!#"
+            }
+          success: '{"status": 200, "bodyPassthrough": true}'
+          error: >
+            {
+              "status":
+              #! if eq .Response.Body.errorType "InvalidInput" !# 400,
+              #! else !#
+              #! if eq .Response.Body.errorType "NotFound" !# 404,
+              #! else !# 500,
+              #! end !#
+              #! end !#
+              "bodyPassthrough": true
+            }
+  /v1/cloud/resourceid/{resourceid}:
```

Comment:
nit: path as per REST `/v1/cloud/resource/{resourceid}`  It looks like current implementation pattern is similar to arnid.

Reply:
FWIW we are not conforming to REST standards. 

In this case, we want to be explicit that the endpoint is for resource id based lookups so I am leaning towards keeping it the same.

Reply:
`resource` is too generic as we have separate endpoints for each lookup type, in this case, this is a resource id based lookup. If we will have a generic `resource` lookup, then the parameter should allow them to choose between ip, hostname or resource id but that would be a significant change.

Reply:
sounds good, i remember we had similar discussion offline.